app/src/main/AndroidManifest.xml

```diff
@@ -34,15 +34,6 @@
                 <category android:name="android.intent.category.DEFAULT" />
                 <category android:name="android.intent.category.BROWSABLE" />
 
-                <data
-                    android:scheme="https"
-                    android:host="@string/FRONTEND_HOST"/>
-                <data
-                    android:pathPrefix="/e"/>
-                <data
-                    android:pathPrefix="/reset-password"/>
-                <data
-                    android:pathPrefix="/email"/>
```

Comment:
?

Reply:
This just got added up from my repository unknowingly So i removed it .

Reply:
Revert it!

Reply:
> This just got added up from my repository unknowingly So i removed it .

I wish magic existed in real world, but nothing gets aded unknowingly

Reply:
Done :+1: